extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php

```diff
@@ -261,15 +261,15 @@ function wfDevBoxLogExceptions( $errorText ) {
  * Example: muppet.wikia.com -> muppet.wikia.com
  */
 function getForcedWikiValue(){
-	global $wgDevelEnvironmentName;
+	global $wgDevDomain;
 
-	if (!isset($_SERVER['HTTP_HOST'])) {
+	if ( !isset( $_SERVER['HTTP_HOST'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "HTTP_HOST" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)